### ISSUE TITLE
Ethics_AI_Workshop

### ISSUE THREAD

**Issue by carkopf** [FZJ-INM1-BDA/HAICon2024-satellite-events#1]
# Title: Ethics of Artificial Intelligence in Research Contexts

# Responsible person(s)
Professor Dr. Bert Heinrichs (b.heinrichs@fz-juelich.de), Dr. PD Jan-Hendrik Heinrichs (j.heinrichs@fz-juelich.de), and Dr. Charles Rathkopf (c.rathkopf@fz-juelich.de)

AG Neuroethics and Ethics of AI, 
Institute for Neuroscience and Medicine 7: Brain and Behaviour

# Format
Workshop

# Timeframe
¼ day (about 3.5-4 h)

# Description
The workshop will discuss current issues in the ethics of Artificial Intelligence in research Contexts. Topics will include the ethical dimensions of AI-systems in general as well as particular issues arising in specialized contexts of application and particular types of AI-systems. Amongst them:
- Dual use potential
- AI in medical diagnostics
- use of unethical / illegal training data

Expert sessions will comprise opening presentations and opportunity for detailed discussion. 

# Requirements
Teaching room with presentation equipment.
Minimum 10, maximum 40.

**Comment by SusanneWenzel** [comment]
@carkopf thank you for this submission. That sounds very interesting and highly relevant to the community! Can you add some more details about the setup of the workshop? Do you plan to open a call for participation, do you invite speakers, or do you rather plan for an introductory talk followed by an open discussion? If already known, who will be the speakers?

**Comment by carkopf** [comment]
We would indeed like to open a call for participation, with the aim of inviting putting together 6 short presentations (20 minutes talking, 10 minutes Q&A), plus a break and a roundtable discussion. It is likely that one of us (Bert, Hendrik, or me) will give one of the talks, but otherwise we are open. As long as you confirm that you think the open call is appropriate, we'll circulate it right away.

**Comment by SusanneWenzel** [comment]
Yes, I think that's a good idea. It would allow external people to contribute. If you send us the call, the H.AI outreach team in Munich can advertise it with the Conf announcements. We will anyway advertise all satellite events on the Conf webpage and with the overall registration. However, I guess we will have separate registrations for the satellite events anyway. I'm happy to help with that.

**Comment by carkopf** [comment]
Ok excellent. I'll write up the call tomorrow, after consulting with Bert and Hendrik, and then I'll post it for you here. Thanks very much for the help.

**Comment by carkopf** [comment]
Hi Susanne - ok I've got a simple PDF version of the call for papers. 
[Helmholtz_AIEthics.pdf](https://github.com/FZJ-INM1-BDA/HAICon2024-satellite-events/files/13989240/Helmholtz_AIEthics.pdf)

**Comment by carkopf** [comment]
I'd be happy to have your help in setting up registration, when the time comes. I suppose I'll need to organize coffee too. Can't have a conference without caffeine. In the meantime, I'll send the call around to a couple of German ethics lists and spread the word.

**Comment by carkopf** [comment]
OK here is an updated version with (i) "call for contributions" and (ii) an explicit message that talks will be given in-person on the Jülich campus.  (Given the small size, we would prefer a fully in-person setup). 
[Helmholtz_AIEthics.pdf](https://github.com/FZJ-INM1-BDA/HAICon2024-satellite-events/files/13990485/Helmholtz_AIEthics.pdf)

**Comment by SusanneWenzel** [comment]
@carkopf do you have a schedule (talk titles, abstracts) for the workshop? I'd like to add it to the website

**Comment by carkopf** [comment]
Got a schedule for you. Here it is: 

13 - 13:45: Hendrik Heinrichs (Jülich). "Brain age Prediction and the Challenge of Biological Concepts of Aging"
13:45 - 14:30: Rooke Christy. (Johns Hopkins) "Blackbox AI in Healthcare: How an Attitude of Pursuit can Mitigate Epistemic and Ethical concerns" 
14:30 - 14:45: break 
14:45 - 15:30: Florian Richter. (Ingolstadt) "AI Ethics as a Form of Research Ethics"
15:30 - 16:15: Theresa Willem. (Technical University of Munich) "Suspended Responsibility. How agential, temporal, and local shifts suspend responsibility for decideions supported by machine learning"
16:15 - 17: Charles Rathkopf. (Jülich) "Foundation models in science: some risks and opportunities"

**Comment by SusanneWenzel** [comment]
Thanks. Added to the website: https://events.hifis.net/event/1403/page/286-workshop-on-ethics-of-artificial-intelligence-in-research-contexts

**Comment by carkopf** [comment]
Thank you!
